xivo/wsgi/__init__.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import time
+
+from cheroot import wsgi
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPServer_patch(wsgi.Server):
```

Comment:
I would add a comment explaining that this solves a bug, ideally linking to a JIRA ticket(if the ticket is a WAZO ticket in the public JIRA).

Reply:
Updated